guides/runtime-parser.md

```diff
@@ -0,0 +1,308 @@
+# Introduction
+
+*This guide assumes that the [App Shell guide](https://mobile.angular.io/guides/app-shell.html) has been completed.*
+
+Now we're going to take a look at another part of the Angular Mobile Toolkit - the App Shell Runtime Parser. The Runtime Parser is a library which has the following features:
+
+- Automatically generates the App Shell of our application by using a predefined template.
```

Comment:
This doesn't clearly describe what the parser does. Should make clear that the shell is extrapolated from a fully dynamic prerendered page (though the words "dynamic" and "prerendered" aren't necessarily the most meaningful words to drive the point home). The sentence as it reads now sounds like it's describing the pre-renderer instead of the parser.